Documentation/gettingstarted/bird.rst

```diff
@@ -0,0 +1,307 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _bird:
+
+****************************
+Using BIRD to run BGP
+****************************
+
+`BIRD is an open-source implementation for routing Internet Protocol
+packets on Unix-like operating systems <https://en.wikipedia.org/wiki/Bird_Internet_routing_daemon>`_.
+If you are not familiar with it, you had best have a glance at the `User's Guide
+<https://bird.network.cz/?get_doc&f=bird.html&v=20>`_ first.
+
+`BIRD <https://bird.network.cz>`_ maintains two release families at present:
+``1.x`` and ``2.x``, and the configuration format varies a lot between them.
+Unless you have already deployed the ``1.x``, we suggest using ``2.x``
+directly, as the ``2.x`` will live longer. In the following, we will denote
+``bird`` as the ``bird2`` software.
+
+This guide shows how to install and configure bird on CentOS `7.x` to make it
+collaborating with Cilium. Installation and configurations on other platforms
+should be very similar.
+
+Install bird
+##################
+
+.. code:: bash
+
+    $ yum install -y bird2
+
+    $ systemctl enable bird
+    $ systemctl restart bird
+
+
+Test the installation:
+
+.. code:: bash
+
+    $ birdc show route
+    BIRD 2.0.6 ready.
+
+    $ birdc              # interactive shell
+    BIRD 2.0.6 ready.
+    bird> show bfd sessions
+    There is no BFD protocol running
+    bird>
+    bird> show protocols all
+    Name       Proto      Table      State  Since         Info
+    device1    Device     ---        up     10:53:40.147
+
+    direct1    Direct     ---        down   10:53:40.147
+      Channel ipv4
+        State:          DOWN
+        Input filter:   ACCEPT
+        Output filter:  REJECT
+    ...
+
+Basic configuration
+#####################
+
+It's hard to discuss bird configurations without considering specific BGP
+schemes. However, BGP scheme design is beyond of the scope of this guide.
+If you are interested in this topic, refer to `BGP in the Data Center
+<https://www.oreilly.com/library/view/bgp-in-the/9781491983416/>`_ (O'Reilly,
+2017) for a quick start.
+
+In the following, we will restrict our BGP scenario as follows:
+
+.. image:: images/bird_sample_topo.png
+   :scale: 35%
+
+* physical network: simple 3-tier hierarchical architecture
+* nodes connect to physical network via layer 2 switches
+* announcing each node's PodCIDR to physical network via `bird`
+* for each node, do not import route announcements from physical network
+
+In this design, the BGP connections look like this:
+
+.. image:: images/bird_sample_bgp.png
+   :scale: 35%
+
+This scheme is simple in that:
+
+* core routers learn PodCIDRs from `bird`, which makes the Pod IP addresses
```

Comment:
You have several occurrences of “bird” between single backquotes in the document, should there be double-backquotes instead?

Reply:
sorry, unintentionally treated this as markdown, i'll fix them.